tensorflow_probability/examples/models/bayesian_resnet.py

```diff
@@ -0,0 +1,104 @@
+"""Builds a Bayesian ResNet18 Model"""
+
+import tensorflow as tf
+import tensorflow_probability as tfp
+
+
+def bayesian_resnet(input_dim,
+                    num_classes=10,
+                    kernel_posterior_scale_mean=-9,
+                    kernel_posterior_scale_stddev=0.1,
+                    kernel_posterior_scale_constraint=0.2):
+  """
+  Constructs a ResNet18 model for training.
+
+  Args:
+    input_dim: A `tuple` indicating the Tensor shape.
```

Comment:
rename to `input_shape`. dim refers to a single value (i.e., one element of a Tensor's rank).